Code for populating the users tables

sartkap opened the pull request:
As of now, the data regarding the club positions and facebook username has not been scraped.

tanvinerkar committed:
image scraping added

ditttu commented on users/jobs/sample.py:
```diff
@@ -0,0 +1,9 @@
+from django_extensions.management.jobs import BaseJob
+
+
+class Job(BaseJob):
```
Why is this present?

ditttu commented on users/jobs/daily/scrape.py:
```diff
@@ -0,0 +1,123 @@
+from bs4 import BeautifulSoup
+import re
+import requests
+import sqlite3
```
Why SQLite, not Postgres?

ditttu commented on users/jobs/daily/test1.py:
```diff
@@ -0,0 +1,135 @@
+from bs4 import BeautifulSoup
+import re
+import requests
+import psycopg2
```
Why 2 files?

ditttu commented on lifeiitkbackend/settings.py:
```diff
@@ -76,8 +77,11 @@
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'students',
+        'USER': 'aditya',
+        'PASSWORD': 'thisisasecret',
```
Don’t add directly, use secret.yaml

ditttu commented:
Also, squash commits

sartkap committed:
parent 0e703e49525f88057f47e1eb55ccfa2d1d227055
author sartkap <sarthakkapoor76@gmail.com> 1559567903 +0530
committer sartkap <sarthakkapoor76@gmail.com> 1559582795 +0530

conflicts resolved

sartkap committed:
Merge branch 'master' into master
